specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/AnomalyDetector.json

```diff
@@ -106,41 +106,1303 @@
+        "x-ms-examples": {
+          "Detect anomaly status of the latest point in time series example": {
+            "$ref": "./examples/LastDetect.json"
+          }
+        }
+      }
+    },
+    "/timeseries/changepoint/detect": {
+      "post": {
+        "summary": "Detect change point for the entire series",
+        "description": "Evaluate change point score of every series point",
+        "operationId": "ChangePointDetect",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "description": "Time series points and granularity is needed. Advanced model parameters can also be set in the request if needed.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ChangePointDetectRequest"
+            }
+          }
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful operation.",
+            "schema": {
+              "$ref": "#/definitions/ChangePointDetectResponse"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/APIError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Detect change point example": {
+            "$ref": "./examples/ChangePointDetect.json"
+          }
+        }
+      }
+    },
+    "/timeseries/{timeSeriesId}": {
+      "get": {
+        "summary": "Get meta information of the specified timeseries.",
+        "description": "corresponds to create series, get series meta with timeseries id.",
+        "operationId": "TimeSeries_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/timeSeriesId"
+          }
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful operation.",
+            "schema": {
+              "$ref": "#/definitions/TimeSeries"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/APIError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get information of a time series example": {
+            "$ref": "./examples/TimeSeriesGet.json"
+          }
+        }
+      },
+      "post": {
+        "summary": "Creat a timeseries.",
+        "description": "Create series, user need to provide timeSeriesId and granularity, if the series has dimension or name, description can be provided through the interface.",
+        "operationId": "TimeSeries_Create",
+        "parameters": [
+          {
+            "$ref": "#/parameters/timeSeriesId"
```

Comment:
When user create a time series with same body should return 204, if it's a successful call, it should return 201

Reply:
Resolved